src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -902,13 +918,24 @@ def main():
     optional = parser.add_argument_group('optional arguments')
 
     # Required flag arguments
-    required.add_argument(f'--{loader_common_names.LOAD_VER_ARG_NAME}', required=True, type=str,
-                          help=loader_common_names.LOAD_VER_DESCR)
-
     required.add_argument(f'--{loader_common_names.KBASE_COLLECTION_ARG_NAME}', required=True, type=str,
                           help=loader_common_names.KBASE_COLLECTION_DESCR)
 
+    required.add_argument(f'--{loader_common_names.SOURCE_VER_ARG_NAME}', required=True, type=str,
+                          help=loader_common_names.SOURCE_VER_DESCR)
+
     # Optional arguments
+    optional.add_argument(
+        f"--{loader_common_names.ENV_ARG_NAME}",
+        type=str,
+        choices=loader_common_names.KB_ENV + [loader_common_names.DEFAULT_ENV],
+        default='PROD',
+        help="Environment containing the data to be processed. (default: PROD)",
+    )
+
+    optional.add_argument(f'--{loader_common_names.LOAD_VER_ARG_NAME}', type=str,
+                          help=loader_common_names.LOAD_VER_DESCR + ' (defaults to the source version)')
```

Comment:
What happens if someone makes a mistake and forgets to enter the correct new load_ver, and the old load_ver that == source_ver already exists?

Reply:
* if load_ver == source_ver does exist, it will just overwrites existing result jsonl files with exactly the same content. 
* if loand_ver == source_ver doesn't exist, it will error due to result dir cannot be found. 
We should be fine in either situation.

Reply:
> if load_ver == source_ver does exist, it will just overwrites existing result jsonl files with exactly the same content.

Ok, you're right for the parser. What about for the tool runner and tool images?

Reply:
if load_ver == source_ver already exists, tool runner will warn user and user has to use --force.
if load_ver == source_ver doesn't exist, the tool runner will run and create a source_ver dir in the collectionsdata dir.

Reply:
:+1: